src/libraries/System.Net.NameResolution/src/System/Net/Dns.cs

```diff
@@ -39,15 +38,11 @@ public static IPHostEntry GetHostEntry(IPAddress address)
                 throw new ArgumentException(SR.Format(SR.net_invalid_ip_addr, nameof(address)));
             }
 
-            IPHostEntry ipHostEntry = GetHostEntryCore(address);
-
-            if (NetEventSource.IsEnabled) NetEventSource.Exit(address, $"{ipHostEntry} with {ipHostEntry.AddressList.Length} entries");
```

Comment:
I would leave it in as Info so we can know how names were resolved.

Reply:
I'm putting it back as Info per your request, but this is a great example of how inconsistent the current logging is: while this overload logs the result, many of the others (including most of the async ones) don't.

Reply:
yes, there is lot of inconsistencies. I'm wondering if we should proceed and open separate issue for each area to add what we think is useful. It may be more work but we may end up with better output.

Reply:
> I'm wondering if we should proceed and open separate issue for each area to add what we think is useful.

That's fine.  Honestly I'd be fine deleting all of inherited tracing and starting over.